### PR TITLE
[AIRFLOW-4163] IntervalCheckOperator supports relative diff and not i…

### PR DESCRIPTION
**Pull request by milton0825**
Make `IntervalCheckOperator` supports calculating ratio with relative difference. And also add an option to not ignore zero values in metric.

Cherry picked from: https://github.com/apache/airflow/pull/4983

### CODE REVIEW

**Comment by milton0825** [comment]
@govindkothari @astahlman @feng-tao

**Comment by feng-tao** [comment]
👍

File: airflow/operators/check_operator.py [hunk 1]
```diff
@@ -226,25 +259,37 @@ def execute(self, context=None):
         rlog = "Ratio for {0}: {1} \n Ratio threshold : {2}"
         fstr = "'{k}' check failed. {r} is above {tr}"
         estr = "The following tests have failed:\n {0}"
-        countstr = "The following {j} tests out of {n} failed:"
+        countstr = "The following {j} tests out of {n} fai:qled:"
```
Comment (feng-tao) [inline airflow/operators/check_operator.py]: typo?

Comment (milton0825) [reply airflow/operators/check_operator.py]: oh man

Comment (milton0825) [reply airflow/operators/check_operator.py]: looks like I was quitting vim

**Pull request merged**